### PR TITLE
Fix crash on closing a reopenend roll-call with be2

### PR DESCRIPTION
**Pull request by Warhedgehog**
Fix to #1072

### CODE REVIEW

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/repository/remote/LAONetworkManager.java [hunk 1]
```diff
@@ -81,7 +84,13 @@ private void resubscribeToChannelOnReconnection() {
             .subscribe(
                 event ->
                     subscribedChannels.forEach(
-                        channel -> disposables.add(subscribe(channel).subscribe()))));
+                        channel ->
+                            disposables.add(
+                                subscribe(channel)
+                                    .subscribe(
+                                        () -> Log.d(TAG, "nothing to see"),
+                                        error -> Log.d(TAG, "error here too")))),
```
Comment (Tesa27) [inline fe2-android/app/src/main/java/com/github/dedis/popstellar/repository/remote/LAONetworkManager.java:92]: should we maybe put some more explicit errors?

**Review by Tesa27 — CHANGES_REQUESTED** [review]

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/repository/remote/LAONetworkManager.java (hunk 1 above)
Comment (Warhedgehog) [reply fe2-android/app/src/main/java/com/github/dedis/popstellar/repository/remote/LAONetworkManager.java:92]: Might help indeed :)

**Review by Tesa27 — APPROVED** [review]
LGTM

**Review by jbsv — APPROVED** [review]
more logs 👍

**Review by zoemarin — COMMENTED** [review]
looks fine to me

**Pull request merged**